src/codeu/chat/server/Model.java

```diff
@@ -148,11 +149,75 @@ public void changeAuthority(ConversationHeader conversation, Uuid targetUser, by
     userAuthority.put(targetUser, authorityByte);
   }
 
-  public boolean validateAuthority(ConversationHeader conversation, Uuid targetUser, byte authorityByte)
+  public boolean isBannedUser(ConversationHeader conversation,Uuid targetUser)
+  {
+    LinkedHashMap<Uuid, Byte> userAuthority = authority.get(conversation.id);
+    Byte authoritByte = userAuthority.get(targetUser);
```

Comment:
Stupid question: If you made this a `byte` instead of a `Byte`, would the code still work? I think it does but I'm not sure...